source/pdal/pdalc_pipeline.cpp

```diff
@@ -35,6 +35,8 @@
 
 #include "pdalc_pointviewiterator.h"
 
+#undef min
```

Comment:
I recommend leaving #4 open and adding a comment that references that issue.

```suggestion
// TODO Address cause of std::min problems. See https://github.com/PDAL/CAPI/issues/4
#undef min
```